### PR TITLE
Add Landing Page and Home Page

### PR DESCRIPTION
**Pull request by OmKakatkar**
## Add Landing Page and Home Page
* Adds some assets.
* Adds a common css styling file
* Add Landing and Home Page styles

image
image

### CODE REVIEW

**Review by lomesshh — APPROVED** [review]
Clean UI
Suggestion :
Instead of adding image directly you can host them and then their links
It may improve your website performance

Rest all looks good to me

File: pages/Home/home.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+			<p class="text-xhuge">Ani Quizz</p>
+			<input type="search" placeholder="Search Quiz..." class="text-xlg" />
+		</header>
+		<main class="container quiz-main">
+			<h1 class="text-heading text-center quiz-heading">Choose a Category</h1>
+			<div class="flex-container">
+				<article class="card shadow rounded">
+					<a href="#" class="text-black">
+						<div class="quiz-image-container">
+							<img src="../../assets/anime.jpg" alt="" />
```
Comment (rohit8686) [inline pages/Home/home.html]: do not leave alt tags empty

File: pages/Landing/landing.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+				<h1 class="text-heading">Ani Quizz: The ultimate quizz app</h1>
+				<p class="text-huge">
+					Test your knowledge with our huge variety of quizes
+				</p>
+				<a href="../Home/home.html" class="btn bg-blue rounded quiz-hero-button"
+					>Lets Go!!</a
+				>
+			</div>
+			<div class="quiz-hero-container">
+				<img src="../../assets/hero.svg" alt="" />
```
Comment (rohit8686) [inline pages/Landing/landing.html]: here also

**Review by rohit8686 — APPROVED** [review]
Everything looks good, can merge it

**Comment by OmKakatkar** [comment]
> Clean UI Suggestion : Instead of adding image directly you can host them and then their links It may improve your website performance
> 
> Rest all looks good to me

Thanks for the suggestion, Will host the images in the future.

File: pages/Home/home.html (hunk 1 above)
Comment (OmKakatkar) [reply pages/Home/home.html]: Thanks for the suggestion. I will add it.

File: pages/Landing/landing.html (hunk 2 above)
Comment (OmKakatkar) [reply pages/Landing/landing.html]: Thanks for the suggestion. I will add it.

**Pull request merged**